src/components/subheader/components/description/Description.js

```diff
@@ -0,0 +1,48 @@
+// Libraries
+import PropTypes from 'prop-types'
+
+// components
+import DescriptionBoxList from "../descriptionBoxList";
+import DescriptionAdress from "../descriptionAdress";
+
+// css
+import './description.css'
+
+
+function Description(props){
+
+    const { restaurantInfo, address } = props;
+
+    return (
+        <div className="description inline">
+            <DescriptionAdress address={address} />
+            <DescriptionBoxList restaurantInfo={restaurantInfo}/>
+        </div>
+    )
+}
+
+Description.propTypes = {
+    restaurantInfo : PropTypes.object,
+    address : PropTypes.object
+}
+
+Description.defaultProps = {
+    restaurantInfo :  {
+        ratingInfo: {
+          rating: "3.9",
+          ratingCount: "100"
+        },
+        deliveryTime: "40 mins",
```

Comment:
You should check for this part, not sure though, will api only send the time in mins or with time with units.